app/src/main/kotlin/team/ommaya/wequiz/android/intro/phone/PhoneFragment.kt

```diff
@@ -0,0 +1,97 @@
+/*
+ * Designed and developed by "옴마야" Team 2023.
+ *
+ * Licensed under the MIT.
+ * Please see full license: https://github.com/mash-up-kr/WeQuiz-Android/blob/main/LICENSE
+ */
+
+package team.ommaya.wequiz.android.intro.phone
+
+import android.os.Bundle
+import android.view.View
+import android.view.ViewGroup.MarginLayoutParams
+import androidx.fragment.app.activityViewModels
+import androidx.navigation.fragment.findNavController
+import team.ommaya.wequiz.android.R
+import team.ommaya.wequiz.android.base.BaseViewBindingFragment
+import team.ommaya.wequiz.android.databinding.FragmentPhoneBinding
+import team.ommaya.wequiz.android.intro.IntroActivity.Companion.LOG_IN_MODE
+import team.ommaya.wequiz.android.intro.IntroActivity.Companion.SIGN_UP_MODE
+import team.ommaya.wequiz.android.intro.IntroViewModel
+import team.ommaya.wequiz.android.utils.KeyboardVisibilityUtils
+import team.ommaya.wequiz.android.utils.formatTextAsPhoneNumber
+import team.ommaya.wequiz.android.utils.setRequestVerifyCodeButtonEnable
+
+class PhoneFragment : BaseViewBindingFragment<FragmentPhoneBinding>(FragmentPhoneBinding::inflate) {
+    private val introViewModel: IntroViewModel by activityViewModels()
+    private lateinit var keyboardVisibilityUtils: KeyboardVisibilityUtils
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        checkVerificationIsSucceed()
+        initView()
+        initKeyboardVisibilityUtils()
+    }
+
+    private fun checkVerificationIsSucceed() {
+        if (introViewModel.isVerificationSucceed.value) {
+            introViewModel.setVerificationSucceed(false)
+            findNavController().navigate(R.id.action_phoneFragment_to_joinFragment)
+        }
```

Comment:
```suggestion
with(introViewModel) {
            if (isVerificationSucceed.value) {
                setVerificationSucceed(false)
                findNavController().navigate(R.id.action_phoneFragment_to_joinFragment)
            }
        }
```
말하면서 어떻게 써야할지 궁금했다던 with 의 간단한 예시입니다~